egs/fame/s5/local/nnet/run_dnn.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
```

Comment:
If you modified the file, add your (c),
if you are using it as it was, please add a note from what recipe this file was taken from